src/doubleTrouble.js

```diff
@@ -26,7 +26,13 @@
  * @return {number[]} - array of remained numbers
  */
 function doubleTrouble(array, target) {
-  // write code here
+  for (let i = 0; i <= array.length; i++) {
+    if (array[i] + array[ i - 1 ] === target) {
+      array.splice([i], 1);
```

Comment:
The first argument of splice should be a number [MDN - Array.prototype.splice](https://developer.mozilla.org/uk/docs/Web/JavaScript/Reference/Global_Objects/Array/splice)